snapcraft/internal/lxd/_containerbuild.py

```diff
@@ -68,6 +69,8 @@ def __init__(self, *, output, source, project_options,
 
         if not remote:
             remote = _get_default_remote()
+        elif remote == 'multipass':
+            remote = self._get_multipass_remote()
```

Comment:
Why do you update the remote variable? I see that the returned value is 'multipass', so it's unchanged. Maybe this shouldn't be called _get_multipass_remote, but something like _setup_multiplass_remote.